src/bandersnatch/tests/test_verify.py

```diff
@@ -31,24 +33,21 @@ def some_dirs(*args: Any, **kwargs: Any) -> List[str]:
     return ["/data/pypi/web/json/bandersnatch", "/data/pypi/web/json/black"]
 
 
-class FakeArgs:
-    delete = True
-    dry_run = True
-    workers = 2
-
-
 class FakeConfig:
     def get(self, section: str, item: str) -> str:
         if section == "mirror":
             if item == "directory":
                 return "/data/pypi"
             if item == "master":
-                return "https://pypi.org/simple/"
+                return "https://unittest.org"
         return ""
 
     def getfloat(self, section: str, item: str, fallback: float = 0.5) -> float:
         return 0.5
 
+    def getint(self, section: str, item: str, fallback: int = 5) -> float:
+        return 5
```

Comment:
Maybe I / someone also missed we should use fallback ?
```suggestion
        return fallback

    def getint(self, section: str, item: str, fallback: int = 5) -> float:
        return fallback
```